core/src/main/java/io/kestra/core/docs/ClassPluginDocumentation.java

```diff
@@ -14,6 +15,7 @@
 @EqualsAndHashCode
 @ToString
 public class ClassPluginDocumentation<T> extends AbstractClassDocumentation<T> {
+    private static final Map<PluginDocIdentifier, ClassPluginDocumentation<?>> CACHE = new HashMap<>();
```

Comment:
```suggestion
    private static final Map<PluginDocIdentifier, ClassPluginDocumentation<?>> CACHE = new ConcurrentHashMap<>();
```

Or you may have concurrency issue as this cache can be access concurrently